src/dapps/RegisterDomain/containers/EnsBidContainer/EnsBidContainer.vue

```diff
@@ -293,6 +295,14 @@ export default {
       this.parseRaw(newVal);
     }
   },
+  mounted() {
+    /*
```

Comment:
Uncomment this since this is necessary when users load in from the interface's saved path.